exp/addons/controllers/clusterresourceset_controller_test.go

```diff
@@ -292,21 +292,18 @@ metadata:
 			},
 			Data: map[string]string{},
 		}
```

Comment:
We're already trying to wait until a first reconciliation is done in l.276-l.286. Unfortunately this doesn't seem to be enough. If there are other/better ways to wait until everything settles, please let me know.